app/tests/test_browse.py

```diff
@@ -0,0 +1,88 @@
+from flask.testing import FlaskClient
+
+from app.tests.mock_database import create_two_test_files
+
+
+def test_load_browse_page(client: FlaskClient):
+    """
+    Given a user accessing the search page
+    When they make a GET request
+    Then they should see the search form and page content.
+    """
+    response = client.get("/browse")
+
+    assert response.status_code == 200
+    assert b"Search for digital records" in response.data
+
+
+def test_submit_search_query(client: FlaskClient):
+    """
+    Given a user accessing the search page
+    When they make a POST request
+    Then they should see the search form and page content.
+    """
+    create_two_test_files()
+
+    response = client.get("/browse")
+    query = "test"
+    response = client.post("/browse", data={"query": query})
+
+    assert response.status_code == 200
+    assert b"Search for digital records" in response.data
+    assert b"Records found 2" in response.data
+    assert b"test series1" in response.data
+    assert b"test series2" in response.data
+
+
+def test_browse_series(client: FlaskClient):
+    """
+    Given a user accessing the search page
+    When they make a GET request with a series id
+    Then they should see the series page content.
+    """
+    files = create_two_test_files()
+    file = files[0]
+    series_id = file.file_consignments.SeriesId
+
+    response = client.get(f"/browse?series={series_id}")
+
+    assert response.status_code == 200
+    assert b"Search for digital records" in response.data
+    assert b"Records found 1" in response.data
+    assert b"test body1" in response.data
+
+
+def test_browse_consignment(client: FlaskClient):
```

Comment:
the consignment page is not currently working properly, but I guess its fine, we know we need to update the page and then we will have to update this test when we do that